Modules/EMCAL/src/ClusterTask.cxx

```diff
@@ -113,36 +137,59 @@ void ClusterTask::startOfCycle()
 
 void ClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
+  std::string InputCell = "emcal-cells";
+  std::string InputCellTR = "emcal-cellstriggerecords";
 
-  std::string InputCell, InputCluster, InputCellIndex;
-  std::string InputCellTrig, InputClusterTrig, InputCellIndexTrig;
+  auto Cell = ctx.inputs().get<gsl::span<o2::emcal::Cell>>(InputCell.c_str());
+  auto CellTR = ctx.inputs().get<gsl::span<o2::emcal::TriggerRecord>>(InputCellTR.c_str());
```

Comment:
Based on the O2 naming rules all variable names must start with a lower case letter

Reply:
I agree and have implemented the changes.